js/Readium.js

```diff
@@ -41,7 +41,7 @@ define(['text!version.json', 'jquery', 'underscore', 'readium_shared_js/views/re
 
             var scripts = "<script type=\"text/javascript\">(" + injectedScript.toString() + ")()<\/script>";
 
-            if (_options && _options.mathJaxUrl && contentDocumentHtml.indexOf("<math") >= 0) {
+            if (_options && _options.mathJaxUrl && contentDocumentHtml.search(/<(\w+:|)(?=math>)/) >= 0) {
```

Comment:
I think that the closing angle bracket `>` after the `math` element name (namespace-prefixed, or not) will break support for MathML attributes in the root element.
Other than that, I think this is ready to merge.